meshes/components/external_antenna.dae

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I wouldn't place it in the components directory, since we got the separated repo for components that exist.
If it's an integral part of Panther I'd just place it on the same level as the body and wheels. Otherwise, maybe we should move it to the components repo.